geode-core/src/test/java/org/apache/geode/internal/cache/ha/HASlowReceiverDUnitTest.java

```diff
@@ -19,7 +19,9 @@
 
 import java.net.SocketException;
 import java.util.Properties;
+import java.util.concurrent.TimeUnit;
 
```

Comment:
I think now we can remove import 
import org.apache.geode.test.dunit.Wait;
import org.apache.geode.test.dunit.WaitCriterion;

From the test file